netbox/templates/ipam/ipaddress.html

```diff
@@ -159,7 +160,18 @@ <h1>{% block title %}{{ ipaddress }}{% endblock %}</h1>
 	<div class="col-md-8">
         {% include 'panel_table.html' with table=parent_prefixes_table heading='Parent Prefixes' %}
         {% if duplicate_ips_table.rows %}
-            {% include 'panel_table.html' with table=duplicate_ips_table heading='Duplicate IP Addresses' panel_class='danger' %}
+            {# Custom version of panel_table.html #}
+            <div class="panel panel-danger">
+                <div class="panel-heading">
+                    <strong>Duplicate IP Addresses</strong>
+                    {% if more_duplicate_ips %}
+                    <div class="pull-right">
+                        <a href="{% url 'ipam:ipaddress_duplicates' pk=ipaddress.pk %}">View all&hellip;</a>
```

Comment:
We try to stay away from legacy HTML elements since they can be difficult to read. I'd recommend using a FontAwesome icon (e.g. `<i class="fa fa-arrow-right"></i>`) or just omitting the ellipsis.

Reply:
Changed it to a button:

![image](https://user-images.githubusercontent.com/5603551/94955048-fe7c0900-04b7-11eb-914c-1183214d2b71.png)